store.go

```diff
@@ -84,6 +84,20 @@ type ApplyStagedLayerOptions struct {
 	DiffOptions *drivers.ApplyDiffWithDifferOpts // Mandatory
 }
 
+// StoreListOptions contains options to pass to MultiList
+type StoreListOptions struct {
```

Comment:
If it is `MultiList`, then `MultiListOptions` (or `StoreMultiListOptions`, but that’s getting unergonomic).

Similarly for the result type — and that one is typically not going to be named by callers, so a long name doesn’t hurt.